### PR TITLE
Adds ability to handle keyboard shortcuts. 

### PR DESCRIPTION
**Pull request by sdee**
Currently maps right arrow to 'NEXT_QUESTION' and up arrow to 'SHOW_ANSWER.'

I used react-hotkeys (https://github.com/chrisui/react-hotkeys) and built a listener reducer and corresponding container/component. 

The challenging part was automatically setting focus to the cards (so you don't have to click on the card before using the shortcuts). I could not get the built-in autoFocus property to work and needed to use both .focus() and tabIndex to set the focus the first time (after loading). It seems like setting focus is more straightforward to input elements vs generic divs. 

The keyboard shortcuts still move the outer element, causing everything to move with the arrow presses. I'd be open to ideas for fixing this.

### CODE REVIEW

**Comment by sdee** [comment]
Also, this will be easy to map to the 'back' functionality once it's implemented.

File: client/src/reducers/filter.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const filter = (state = {}, action) => {
 	case SET_FILTER: {
 		const newState = Object.assign({}, state, {});
 		newState[action.filter] = action.status;
-		return newState;
+		return state;
```
Comment (jagatfx) [inline client/src/reducers/filter.js]: @sdee this is a bug that breaks all filtering. I will revert.

**Comment by jagatfx** [comment]
@sdee since you added the FLIP_CARD action, I think the up key should also trigger flipCard() instead of showAnswer() like clicking does.

**Comment by sdee** [comment]
Agreed. I was wondering if there's a reason to have SHOW_ANSWER (and SHOW_QUESTION which I had added) instead of just having a toggle.

**Pull request merged**